test/jasmine/tests/axes_test.js

```diff
@@ -1796,6 +1796,23 @@ describe('Test axes', function() {
                 'input range, ' + msg + ': ' + ax.range);
         }
 
+        it('can change _inputDomain on call to react when template is present', function(done) {
```

Comment:
I would describe this in a more result-oriented way, something more like

```suggestion
        it('can change axis domain on call to react when template is present', function(done) {
```